projects/angular/src/layout/nav/nav-level.ts

```diff
@@ -184,7 +184,7 @@ export class ClrNavLevel extends FocusTrap implements OnInit {
     while (current) {
       if (current === navHost) {
         return;
-      } else if (current.classList.contains('nav-link')) {
+      } else if (current.classList.contains('nav-link') && this._document.body.clientWidth < LARGE_BREAKPOINT) {
```

Comment:
This logic also exists in the [ngAfterViewInit event](https://github.com/vmware-clarity/ng-clarity/blob/f1e17c83f335029ea46b6dfdf1f0041c10e902ef/projects/angular/src/layout/nav/nav-level.ts#L112), so it could make sense to have a shared function that only closes on small screens? So the logic doesn't have to be duplicated if it needs to go somewhere else?

Reply:
I'm pretty sure these functions are going to change when the @cds/core v6 is released cause we'll have to adjust the code to use the new focus trap otherwise the compiler will start to complain that it can't find some classes that are provided from core (they are in >5.6.0  <6.0.0)